packages/demo-app/package.json

```diff
@@ -27,7 +27,7 @@
     "copy-webpack-plugin": "^6.0.3",
     "html-webpack-plugin": "^4.5.0",
     "node-fetch": "^2.6.0",
-    "source-map-loader": "^2.0.0",
+    "source-map-loader": "^1.0.2",
```

Comment:
I'm reverting this because currently, the Storybook is not working (the source map loader package is breaking the build). The PR that broke it is this one [#781](https://github.com/mui-org/material-ui-x/pull/781)